### PR TITLE
App title, open graph, and descriptions update

### PR DESCRIPTION
**Pull request by ux-git**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,10 +15,17 @@
...
     />
     <meta name="theme-color" content="#f8f9ffff" />
+    <title>Organized: App for Jehovah's Witnesses</title>
     <meta
       name="description"
-      content="A universal app to simplify management of Jehovah's Witnesses congregation life. View assignments, schedule meetings, send reports effortlessly and way more. Simplify your routine tasks and focus on what matters most!"
-    />
+      content="A modern all-in-one app for Jehovah's Witnesses to simplify congregation management. Create meeting schedules, view your assignments, create and submit reports, and way more. Save time on routine tasks and focus on other important things!"
+    />
+    <meta property="og:title" content="Organized: The modern all-in-one ppp for Jehovah's Witnesses congregations">
```
[the remark this answers is not in the document]
Comment (rhahao) [reply index.html:23]: @ux-git: could you implement this change as suggested please?

File: index.html [hunk 2]
```diff
@@ -15,10 +15,17 @@
...
+    <title>Organized: App for Jehovah's Witnesses</title>
     <meta
       name="description"
-      content="A universal app to simplify management of Jehovah's Witnesses congregation life. View assignments, schedule meetings, send reports effortlessly and way more. Simplify your routine tasks and focus on what matters most!"
-    />
+      content="A modern all-in-one app for Jehovah's Witnesses to simplify congregation management. Create meeting schedules, view your assignments, create and submit reports, and way more. Save time on routine tasks and focus on other important things!"
+    />
+    <meta property="og:title" content="Organized: The modern all-in-one app for Jehovah's Witnesses congregations">
+    <meta property="og:description" content="User-friendly, secure, and cross-platform. Organized makes managing Jehovah’s Witnesses congregation life easy and efficient! Create schedules, view assignments, and submit reports — quick and simple. This app is a great help for everyone in the congregation. Try it now and see the difference!">
+    <meta property="og:image" content="img/open-graph/organized-app-og-image.jpg">
```
[the remark this answers is not in the document]
Comment (rhahao) [reply index.html:25]: @ux-git: yes it’s correct and no need to changes. The public path will be removed during the build time so it won’t exist in the end. Thanks.

[the remark this answers is not in the document]
Comment (ux-git) [reply index.html:25]: Great

**Review by rhahao — APPROVED** [review]

**Pull request merged**